examples/OneDShallowWaterGeostrophicAdjustment.jl

```diff
@@ -1,184 +1,428 @@
+# # Linear rotating shallow water dynamics
+#
+#md # This example can be run online via [![](https://mybinder.org/badge_logo.svg)](@__BINDER_ROOT_URL__/generated/OneDShallowWaterGeostrophicAdjustment.ipynb). 
+#md # Also, it can be viewed as a Jupyter notebook via [![](https://img.shields.io/badge/show-nbviewer-579ACA.svg)](@__NBVIEWER_ROOT_URL__/generated/OneDShallowWaterGeostrophicAdjustment.ipynb).
+# 
+#
+#
 # This example solves the linear 1D rotating shallow water equations
-# for the depth-integrated velocities u and v and for the
-# surface elevation η in a fluid with constant depth h,
-# gravitational acceleration g and Coriolis parameter f:
+# for the $u(x, t)$, $v(x, t)$ and the surface surface elevation $\eta(x, t)$, 
```

Comment:
for the depth-averaged velocities $u(x, t)$, $v(x, t)$ and the surface elevation $\eta(x, t)$

Reply:
Andre, should have mentioned that I rewrote the equations for the normal u, v, (not depth integrated). Equations now read:
∂u/∂t - fv = -g ∂η/∂x - D
∂v/∂t + fu = - D
∂η/∂t + H ∂u/∂x = - D

Are these OK?

Reply:
Yes, I agree that depth-averaged is better. Looks good.